Dockerfile

```diff
@@ -114,9 +114,10 @@ ENTRYPOINT ["/usr/bin/docker-entrypoint.sh"]
 ######################################################################
 FROM lean AS dev
 
-COPY ./requirements-dev.txt ./docker/requirements-extra.txt /app/
+COPY ./requirements-dev.txt ./docker/requirements* /app/
 
 USER root
 RUN cd /app \
-    && pip install --no-cache -r requirements-dev.txt -r requirements-extra.txt
+    && pip install --no-cache -r requirements-dev.txt -r requirements-extra.txt \
```

Comment:
is there a reason to not just direct users to add extra requirements in `requirements-extra.txt`?

Reply:
Yep, we don't want local users' changes being managed in Git